features/run.sh

```diff
@@ -18,6 +18,8 @@ for endpoint in Embedded Proxy; do
       host="$embedded_host"
       port="$embedded_port"
     fi
-    ./node_modules/.bin/cucumber-js --format progress-bar -p "${protocol}${endpoint}" --world-parameters "{\"host\": \"${host}\", \"port\": \"${port}\"}"
+
+    # profiles are defined in the cucumber.js file at the root of this project
```

Comment:
so the env variables $CUCUMBER_PROXY_HOST, $CUCUMBER_EBEDDED_HOST, etc, are not used anymore ?

Reply:
Good catch, I removed those because I thought they were redundant with the cucumber.js configuration file.
Change reverted.